.goreleaser.yml

```diff
@@ -38,18 +38,37 @@ nfpms:
     formats:
       - deb
       - rpm
+      - archlinux
     bindir: /usr/bin
+    contents:
```

Comment:
Does `bindir` here need to change to `/usr/sbin` ? 

I think if that path is a symlink on any distro it's going to complain, but I'm not sure which distros symlink it.